Packs/PAN-OS/Playbooks/playbook-PAN-OS_-_Block_IP_-_Static_Address_Group_6_10.yml

```diff
@@ -0,0 +1,623 @@
+    taskid: 8a034a0d-ebea-401c-8b9d-e7e708d94fdb
+    type: regular
+    task:
+      id: 8a034a0d-ebea-401c-8b9d-e7e708d94fdb
+      version: -1
+      name: Commit PAN-OS Instance manually
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1440,
+          "y": 1580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 4bc1586a-2526-4146-8bd5-98477fd43863
+    type: playbook
+    task:
+      id: 4bc1586a-2526-4146-8bd5-98477fd43863
+      version: -1
+      name: PAN-OS Commit Configuration v2
+      playbookName: PAN-OS Commit Configuration v2
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 1220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "1_3_#default#": 0.16,
+      "2_3_#default#": 0.17,
+      "5_3_yes": 0.29
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1765,
+        "width": 1370,
+        "x": 450,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: IP
+  value:
+    complex:
+      root: IP
+      filters:
+      - - operator: isExists
+          left:
+            value:
+              simple: IP.Malicious
+            iscontext: true
+      accessor: Address
+      transformers:
+      - operator: uniq
+  required: false
+  description: IP address to block
+  playbookInputQuery:
+- key: AddressGroupName
+  value:
+    simple: Demisto Remediation - Static Address Group
+  required: false
+  description: Static address group name
+  playbookInputQuery:
+- key: AutoCommit
+  value:
+    simple: "No"
+  required: false
+  description: |-
+    This input establishes whether to commit the configuration automatically.
+    Yes - Commit automatically.
+    No - Commit manually.
+  playbookInputQuery:
+- key: IPDescription
+  value:
+    simple: Automatically created using "PAN-OS - Block IP - Static Address Group" playbook in Cortex XSIAM.
+  required: false
+  description: The description of the IP in case it will get created by the playbook.
+  playbookInputQuery:
+inputSections:
+- inputs:
+  - IP
+  - AddressGroupName
+  - AutoCommit
+  - IPDescription
+  name: General (Inputs group)
+  description: Generic group for inputs
+outputSections:
+- outputs: []
+  name: General (Outputs group)
+  description: Generic group for outputs
+outputs: []
+tests:
```

Comment:
In the original yml, there is only one test playbook. 
- PAN-OS - Block IP - Static Address Group Test
please add the correct tp and remove the others.